tools/kessel-debug-container/installer.sh

```
@@ -2,26 +2,28 @@
-curl -fLo /tmp/zed_checksums.txt https://github.com/authzed/zed/releases/download/v${ZED_VERSION}/checksums.txt
+RPM_URLS=(
+  "https://github.com/authzed/zed/releases/download/v${ZED_VERSION}/zed_${ZED_VERSION}_linux_amd64.rpm"
+  "https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/Packages/librdkafka-${LIBRDKAFKA_VERSION}.el9.x86_64.rpm"
+  "https://dl.fedoraproject.org/pub/epel/9/Everything/x86_64/Packages/k/kcat-${KCAT_VERSION}.el9.x86_64.rpm"
+)
 
-# Verify zed checksum
-pushd /tmp
-grep "zed_${ZED_VERSION}_linux_amd64.rpm" zed_checksums.txt | sha256sum -c -
-popd
+for URL in ${RPM_URLS[*]}; do
```

Shellcheck (0.11.0)

[warning] Use "${array[@]}" (with quotes) to prevent whitespace problems.

(SC2048)